custom/enikshay/integrations/ninetyninedots/repeater_generators.py

```diff
@@ -126,6 +126,16 @@ def _update_episode_case(domain, case_id, updated_properties):
     )
 
 
+def _update_person_case(domain, case_id, updated_properties):
```

Comment:
this is exactly the same as `update_episode_case` above. You could make this generic `update_case` and move it into the `case-utils.py`. That way the Nikshay integration isn't coupled to the 99DOTs one in any way.